hashing/hashing.go

```diff
@@ -14,6 +16,7 @@ type Configuration struct {
 
 func DefaultConfiguration() *Configuration {
 	configuration, _ := LookupDefaultConfiguration("SHA256")
+	//configuration, _ := LookupDefaultConfiguration("BLAKE3")
```

Comment:
debugging leftover?

Reply:
not really...

Depending on what the crypto audit report says, I was about to prepare a subsequent PR to switch the default algorithm to BLAKE3 by default ;-)